### ISSUE TITLE
MacOS download 11.10.2 contains the same version notation as version 11.10.1

### ISSUE THREAD

**Issue by suschizu** [Floorp-Projects/Floorp#914]
### Pre-Submission Checklist

- [X] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [X] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [ ] I have tried clearing the browser cache and cookies.
- [ ] I have tried to create a new profile and the issue still occurs.
- [ ] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [ ] I have included relevant screenshots or console outputs.
- [ ] I have checked this problem is not ocurring in the latest version of Firefox ESR.
- [ ] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

the version notation inside the info.plist hasn't changed:
Short Version String unchanged: 115.8.0 == 115.8.0
Bundle Version unchanged: 11524.2.19 == 11524.2.19

### Reproduction Steps

download apps and compare info.plist

### Expected Behavior

version numbers should increment

### Actual Behavior

has the same version as earlier release

### Screenshots

<img width="568" alt="Screenshot 2024-02-20 at 18 57 08" src="https://github.com/Floorp-Projects/Floorp/assets/16004333/cecec7e0-e64c-403f-a3bf-1267dbaa1370">

### Browser Version

Safari 

### Operating System

Ventura 13.6
